packages/conventional-changelog-cli/README.md

```diff
@@ -57,10 +56,33 @@ To fully customize the tool, please checkout [conventional-changelog](https://gi
 
 The reason why you should commit and tag after `conventionalChangelog` is that the CHANGELOG should be included in the new release, hence `gitRawCommitsOpts.from` defaults to the latest semver tag.
 
-If you use `npm version`, it auto tags immediately after changing the version in package.json. In such case, you might want to specify the version manually and generate the changelog before `npm version`.
 
-Please use this [gist](https://gist.github.com/stevemao/280ef22ee861323993a0) to make a release or change it to your needs.
+## Recommended workflow using `npm version`
+Using the npm scripts to our advantage.
+With the following hooks:
+```json
+{
+  "scripts": {
+    "version": "conventional-changelog -p angular -i CHANGELOG.md -s -r 0 && git add CHANGELOG.md"
```

Comment:
Could this be `postversion` for clarity? (Making it obvious that it occurs _after_ the behavior of `version`)

Reply:
It will defy the purpose and will need to create an other commit.
The goal here is to update the version, generate the changelog, then commit all

Reply:
Ah, so taking advantage of the fact that `npm` does not run `git commit` until the `version` script has completed.

Thank you for the clarification.

`postversion` still seems clearer, but taking advantage of `npm`'s behavior for the `version` script is a clear win for usability.